eclair-core/src/main/scala/fr/acinq/eclair/db/DbEventHandler.scala

```diff
@@ -65,49 +74,68 @@ class Auditor(nodeParams: NodeParams) extends Actor with ActorLogging {
           PaymentMetrics.PaymentParts.withTag(PaymentTags.Direction, PaymentTags.Directions.Sent).record(outgoing.length)
         case _: ChannelPaymentRelayed =>
       }
-      db.add(e)
+      auditDb.add(e)
 
-    case e: NetworkFeePaid => db.add(e)
+    case e: NetworkFeePaid => auditDb.add(e)
 
     case e: ChannelErrorOccurred =>
       e.error match {
         case LocalError(_) if e.isFatal => ChannelMetrics.ChannelErrors.withTag(ChannelTags.Origin, ChannelTags.Origins.Local).withTag(ChannelTags.Fatal, value = true).increment()
         case LocalError(_) if !e.isFatal => ChannelMetrics.ChannelErrors.withTag(ChannelTags.Origin, ChannelTags.Origins.Local).withTag(ChannelTags.Fatal, value = false).increment()
         case RemoteError(_) => ChannelMetrics.ChannelErrors.withTag(ChannelTags.Origin, ChannelTags.Origins.Remote).increment()
       }
-      db.add(e)
+      auditDb.add(e)
 
     case e: ChannelStateChanged =>
       // NB: order matters!
       e match {
         case ChannelStateChanged(_, channelId, _, remoteNodeId, WAIT_FOR_FUNDING_LOCKED, NORMAL, Some(commitments: Commitments)) =>
           ChannelMetrics.ChannelLifecycleEvents.withTag(ChannelTags.Event, ChannelTags.Events.Created).increment()
-          db.add(ChannelLifecycleEvent(channelId, remoteNodeId, commitments.capacity, commitments.localParams.isFunder, !commitments.announceChannel, "created"))
+          val event = ChannelLifecycleEvent.EventType.Created
+          auditDb.add(ChannelLifecycleEvent(channelId, remoteNodeId, commitments.capacity, commitments.localParams.isFunder, !commitments.announceChannel, event))
+          channelsDb.updateChannelMeta(channelId, event)
         case ChannelStateChanged(_, _, _, _, WAIT_FOR_INIT_INTERNAL, _, _) =>
+        case ChannelStateChanged(_, channelId, _, _, OFFLINE, SYNCING, _) =>
+          channelsDb.updateChannelMeta(channelId, ChannelLifecycleEvent.EventType.Connected)
```

Comment:
We may want to have this event in the `auditDb` as well, don't we?

Reply:
I don't think so, because the risk of spam is too high imo.